src/models/interfaces/IGroupable.ts

```diff
@@ -0,0 +1,6 @@
+import { UserGroup } from '../entities';
+
+export interface IGroupable {
```

Comment:
I am not 100% sure we need the GraphQL interfaces with our API size (yet).

Reply:
This is not graphql but business logic interface.